CONTRIBUTORS

```diff
@@ -17,3 +17,4 @@ Douglas Stebila (University of Waterloo)
 Goutam Tamvada (University of Waterloo)
 John Underhill
 Sebastian Verschoor (University of Waterloo)
+Michael Baentsch (IBM Research-Zurich)
```

Comment:
Could you place this before "Eric Crockett"? The list is alphabetically sorted by last name.